src/components/Plants/Plant.tsx

```diff
@@ -7,10 +10,21 @@ export type PlantType = {
   isSafe: boolean;
 };
 
+const PHOTOS_TO_DISPLAY = 5;
+
 export default function Plant({ plant }: { plant: PlantType }) {
+  const photos = usePlantPics(plant.name.lat);
+  const urls = getPhotosUrls(photos.slice(0, PHOTOS_TO_DISPLAY));
+
   return (
     <div className="pb-3">
       {plant.name.pl[0]} {plant.isSafe ? '✅' : '🍄'}
+      <div className="flex pt-2 pb-3">
```

Comment:
preview on vercel (https://cats-git-aw-fetch-pics.anna-wro.vercel.app/) has no styles and it looks like it's a tailwind issue.

e.g. https://github.com/vercel/next.js/issues/14286

Reply:
I disabled auto deployment for now